lib/esp/resources/scan_interval.rb

```diff
@@ -1,21 +1,25 @@
 module ESP
   class ScanInterval < ESP::Resource
-    ##
     # The external account the scan interval belongs to
+    #
+    # @return [ESP::ExternalAccount]
     belongs_to :external_account, class_name: 'ESP::ExternalAccount'
 
-    ##
     # The service the scan interval belongs to
+    #
+    # @return [ESP::Service]
     belongs_to :service, class_name: 'ESP::Service'
 
     # Find a Scan Interval by id
     #
-    # ==== Parameter
-    #
-    # +id+ | Required | The ID of the scan interval to retrieve
+    # *call-seq* -> +super.find(id)+
     #
-    # :call-seq:
-    #  find(id)
+    # @overload find(id)
+    #   @param id [Integer, Numeric, #to_i] Required ID of the scan interval to retrieve.
+    # @overload find(non_numeric, options)
```

Comment:
Not sure about documenting this overload, but if we do, it should be @overload find(options = {})